### ISSUE TITLE
Kubelet unable to collect `node_cpu_usage_seconds_total` and `node_memory_working_set_bytes`

### ISSUE THREAD

**Issue by younghoon-klleon** [kubernetes/kubernetes#123049]
### What happened?

Some nodes have kubelet successfully collecting node_cpu_usage_seconds_total and node_memory_working_set_bytes, while on some nodes, it fails to collect them.
![kubelet_error](https://github.com/kubernetes/kubernetes/assets/102716628/aae5d07c-48af-485b-b6bf-c3ec0bfe98bf)

### What did you expect to happen?

All nodes successfully collecting the two metrics.

### How can we reproduce it (as minimally and precisely as possible)?

sorry, Not sure exactly what caused it.

### Kubernetes version

<details>

```console
$ kubectl version
Client Version: version.Info{Major:"1", Minor:"27", GitVersion:"v1.27.7", GitCommit:"07a61d861519c45ef5c89bc22dda289328f29343", GitTreeState:"clean", BuildDate:"2023-10-18T11:42:32Z", GoVersion:"go1.20.10", Compiler:"gc", Platform:"linux/amd64"}
Kustomize Version: v5.0.1
Server Version: version.Info{Major:"1", Minor:"27", GitVersion:"v1.27.7", GitCommit:"07a61d861519c45ef5c89bc22dda289328f29343", GitTreeState:"clean", BuildDate:"2023-10-18T11:33:23Z", GoVersion:"go1.20.10", Compiler:"gc", Platform:"linux/amd64"}
```

</details>

### Cloud provider

<details>
onpremise
</details>

### OS version

<details>

```console
# On Linux:
$ cat /etc/os-release
NAME="Ubuntu"
VERSION="20.04.6 LTS (Focal Fossa)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 20.04.6 LTS"
VERSION_ID="20.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=focal
UBUNTU_CODENAME=focal

$ uname -a
Linux node1 5.4.0-169-generic #187-Ubuntu SMP Thu Nov 23 14:52:28 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux
```

</details>

### Install tools

<details>
kubespray 2.23.1
</details>

### Container runtime (CRI) and version (if applicable)

<details>
containerd v1.7.5
</details>

### Related plugins (CNI, CSI, ...) and versions (if applicable)

<details>

</details>

**Comment by tamilselvan1102** [comment]
please exec below comment and provide the log
kubectl get --raw /apis/metrics.k8s.io/v1beta1/nodes

**Comment by younghoon-klleon** [comment]
@tamilselvan1102 
`{                                                                                                                                                                                                                           
  "kind": "NodeMetricsList",                                                                                                                                                                                                
  "apiVersion": "metrics.k8s.io/v1beta1",                                                                                                                                                                                   
  "metadata": {},                                                                                                                                                                                                           
  "items": [                                                                                                                                                                                                                
    {                                                                                                                                                                                                                       
      "metadata": {                                                                                                                                                                                                         
        "name": "node13",                                                                                                                                                                                                   
        "creationTimestamp": "2024-01-31T10:39:01Z",                                                                                                                                                                        
        "labels": {                                                                                                                                                                                                         
          "beta.kubernetes.io/arch": "amd64",                                                                                                                                                                               
          "beta.kubernetes.io/os": "linux",                                                                                                                                                                                 
          "feature.node.kubernetes.io/cpu-cpuid.ADX": "true",                                                                                                                                                               
          "feature.node.kubernetes.io/cpu-cpuid.AESNI": "true",                                                                                                                                                             
          "feature.node.kubernetes.io/cpu-cpuid.AVX": "true",                                                                                                                                                               
          "feature.node.kubernetes.io/cpu-cpuid.AVX2": "true",                                                                                                                                                              
          "feature.node.kubernetes.io/cpu-cpuid.AVXVNNIINT8": "true",                                                                                                                                                       
          "feature.node.kubernetes.io/cpu-cpuid.CETSS": "true",                                                                                                                                                             
          "feature.node.kubernetes.io/cpu-cpuid.CLZERO": "true",                                              
          "feature.node.kubernetes.io/cpu-cpuid.CMPXCHG8": "true",                                                                                                                                                          
          "feature.node.kubernetes.io/cpu-cpuid.CPBOOST": "true",                                             
          "feature.node.kubernetes.io/cpu-cpuid.EFER_LMSLE_UNS": "true",                                      
          "feature.node.kubernetes.io/cpu-cpuid.FMA3": "true",                                                
          "feature.node.kubernetes.io/cpu-cpuid.FP256": "true",                                               
          "feature.node.kubernetes.io/cpu-cpuid.FSRM": "true",                                                
          "feature.node.kubernetes.io/cpu-cpuid.FXSR": "true",                                                
          "feature.node.kubernetes.io/cpu-cpuid.FXSROPT": "true",
          "feature.node.kubernetes.io/cpu-cpuid.IBPB": "true",                                                
          "feature.node.kubernetes.io/cpu-cpuid.IBRS": "true",                                                
          "feature.node.kubernetes.io/cpu-cpuid.IBRS_PREFERRED": "true",                                      
          "feature.node.kubernetes.io/cpu-cpuid.IBRS_PROVIDES_SMP": "true",
          "feature.node.kubernetes.io/cpu-cpuid.IBS": "true",      
          "feature.node.kubernetes.io/cpu-cpuid.IBSBRNTRGT": "true",                                          
          "feature.node.kubernetes.io/cpu-cpuid.IBSFETCHSAM": "true",                                         
          "feature.node.kubernetes.io/cpu-cpuid.IBSFFV": "true",                                              
          "feature.node.kubernetes.io/cpu-cpuid.IBSOPCNT": "true",                                                                                                                                                          
          "feature.node.kubernetes.io/cpu-cpuid.IBSOPCNTEXT": "true",                                         
          "feature.node.kubernetes.io/cpu-cpuid.IBSOPSAM": "true",                                            
          "feature.node.kubernetes.io/cpu-cpuid.IBSRDWROPCNT": "true",                                        
          "feature.node.kubernetes.io/cpu-cpuid.IBSRIPINVALIDCHK": "true",                                    
          "feature.node.kubernetes.io/cpu-cpuid.IBS_FETCH_CTLX": "true",                                                                                                                                                    
          "feature.node.kubernetes.io/cpu-cpuid.IBS_OPFUSE": "true",                                                                                                                                                        
          "feature.node.kubernetes.io/cpu-cpuid.IBS_PREVENTHOST": "true",                                     
          "feature.node.kubernetes.io/cpu-cpuid.INT_WBINVD": "true",                                          
          "feature.node.kubernetes.io/cpu-cpuid.INVLPGB": "true",                                             
          "feature.node.kubernetes.io/cpu-cpuid.LAHF": "true",                                                
          "feature.node.kubernetes.io/cpu-cpuid.LBRVIRT": "true",                                             
          "feature.node.kubernetes.io/cpu-cpuid.MCAOVERFLOW": "true",                                         
          "feature.node.kubernetes.io/cpu-cpuid.MCOMMIT": "true",
          "feature.node.kubernetes.io/cpu-cpuid.MOVBE": "true",
          "feature.node.kubernetes.io/cpu-cpuid.MOVU": "true",
          "feature.node.kubernetes.io/cpu-cpuid.MSRIRC": "true",
          "feature.node.kubernetes.io/cpu-cpuid.MSR_PAGEFLUSH": "true",
          "feature.node.kubernetes.io/cpu-cpuid.NRIPS": "true",
          "feature.node.kubernetes.io/cpu-cpuid.OSXSAVE": "true",
          "feature.node.kubernetes.io/cpu-cpuid.PPIN": "true",
          "feature.node.kubernetes.io/cpu-cpuid.PSFD": "true",
          "feature.node.kubernetes.io/cpu-cpuid.RDPRU": "true",
          "feature.node.kubernetes.io/cpu-cpuid.SEV": "true",
          "feature.node.kubernetes.io/cpu-cpuid.SEV_64BIT": "true",
          "feature.node.kubernetes.io/cpu-cpuid.SEV_ALTERNATIVE": "true",
          "feature.node.kubernetes.io/cpu-cpuid.SEV_DEBUGSWAP": "true",
          "feature.node.kubernetes.io/cpu-cpuid.SEV_ES": "true",
          "feature.node.kubernetes.io/cpu-cpuid.SEV_RESTRICTED": "true",
          "feature.node.kubernetes.io/cpu-cpuid.SEV_SNP": "true",
          "feature.node.kubernetes.io/cpu-cpuid.SHA": "true",
          "feature.node.kubernetes.io/cpu-cpuid.SME": "true",
          "feature.node.kubernetes.io/cpu-cpuid.SME_COHERENT": "true",
          "feature.node.kubernetes.io/cpu-cpuid.SPEC_CTRL_SSBD": "true",
          "feature.node.kubernetes.io/cpu-cpuid.SSE4A": "true",
          "feature.node.kubernetes.io/cpu-cpuid.STIBP": "true",
          "feature.node.kubernetes.io/cpu-cpuid.STIBP_ALWAYSON": "true",
          "feature.node.kubernetes.io/cpu-cpuid.SUCCOR": "true",
          "feature.node.kubernetes.io/cpu-cpuid.SVM": "true",
          "feature.node.kubernetes.io/cpu-cpuid.SVMDA": "true",
          "feature.node.kubernetes.io/cpu-cpuid.SVMFBASID": "true",
          "feature.node.kubernetes.io/cpu-cpuid.SVML": "true",
          "feature.node.kubernetes.io/cpu-cpuid.SVMNP": "true",
          "feature.node.kubernetes.io/cpu-cpuid.SVMPF": "true",
          "feature.node.kubernetes.io/cpu-cpuid.SVMPFT": "true",
          "feature.node.kubernetes.io/cpu-cpuid.SYSCALL": "true",
          "feature.node.kubernetes.io/cpu-cpuid.SYSEE": "true",
          "feature.node.kubernetes.io/cpu-cpuid.TLB_FLUSH_NESTED": "true",
          "feature.node.kubernetes.io/cpu-cpuid.TOPEXT": "true",
          "feature.node.kubernetes.io/cpu-cpuid.TSCRATEMSR": "true",
          "feature.node.kubernetes.io/cpu-cpuid.VAES": "true",
          "feature.node.kubernetes.io/cpu-cpuid.VMCBCLEAN": "true",
          "feature.node.kubernetes.io/cpu-cpuid.VMPL": "true",
          "feature.node.kubernetes.io/cpu-cpuid.VMSA_REGPROT": "true",
          "feature.node.kubernetes.io/cpu-cpuid.VPCLMULQDQ": "true",
          "feature.node.kubernetes.io/cpu-cpuid.VTE": "true",
          "feature.node.kubernetes.io/cpu-cpuid.WBNOINVD": "true",
          "feature.node.kubernetes.io/cpu-cpuid.X87": "true",
          "feature.node.kubernetes.io/cpu-cpuid.XGETBV1": "true",
          "feature.node.kubernetes.io/cpu-cpuid.XSAVE": "true",
          "feature.node.kubernetes.io/cpu-cpuid.XSAVEC": "true",
          "feature.node.kubernetes.io/cpu-cpuid.XSAVEOPT": "true",
          "feature.node.kubernetes.io/cpu-cpuid.XSAVES": "true",
          "feature.node.kubernetes.io/cpu-hardware_multithreading": "true",
          "feature.node.kubernetes.io/cpu-model.family": "25",
          "feature.node.kubernetes.io/cpu-model.id": "8",
          "feature.node.kubernetes.io/cpu-model.vendor_id": "AMD",
          "feature.node.kubernetes.io/cpu-rdt.RDTCMT": "true",
          "feature.node.kubernetes.io/cpu-rdt.RDTL3CA": "true",
          "feature.node.kubernetes.io/cpu-rdt.RDTMBM": "true",
          "feature.node.kubernetes.io/cpu-rdt.RDTMON": "true",
          "feature.node.kubernetes.io/kernel-config.NO_HZ": "true",
          "feature.node.kubernetes.io/kernel-config.NO_HZ_IDLE": "true",
          "feature.node.kubernetes.io/kernel-version.full": "5.4.0-135-generic",
          "feature.node.kubernetes.io/kernel-version.major": "5",
          "feature.node.kubernetes.io/kernel-version.minor": "4",
          "feature.node.kubernetes.io/kernel-version.revision": "0",
          "feature.node.kubernetes.io/network-sriov.capable": "true",
          "feature.node.kubernetes.io/pci-10de.present": "true",
          "feature.node.kubernetes.io/pci-1a03.present": "true",
          "feature.node.kubernetes.io/pci-8086.present": "true",
          "feature.node.kubernetes.io/pci-8086.sriov.capable": "true",
          "feature.node.kubernetes.io/storage-nonrotationaldisk": "true",
          "feature.node.kubernetes.io/system-os_release.ID": "ubuntu",
          "feature.node.kubernetes.io/system-os_release.VERSION_ID": "20.04",
          "feature.node.kubernetes.io/system-os_release.VERSION_ID.major": "20",
          "feature.node.kubernetes.io/system-os_release.VERSION_ID.minor": "04",
          "feature.node.kubernetes.io/usb-ef_0b05_1984.present": "true",
          "kubernetes.io/arch": "amd64",
          "kubernetes.io/hostname": "node13",
          "kubernetes.io/os": "linux",
          "nvidia.com/cuda.driver.major": "515",
          "nvidia.com/cuda.driver.minor": "86",
          "nvidia.com/cuda.driver.rev": "01",
          "nvidia.com/cuda.runtime.major": "11",
          "nvidia.com/cuda.runtime.minor": "7",
          "nvidia.com/gfd.timestamp": "1704700189",
          "nvidia.com/gpu-driver-upgrade-state": "upgrade-required",
          "nvidia.com/gpu.compute.major": "8",
          "nvidia.com/gpu.compute.minor": "6",
          "nvidia.com/gpu.count": "4",
          "nvidia.com/gpu.deploy.container-toolkit": "true",
          "nvidia.com/gpu.deploy.dcgm": "true",
          "nvidia.com/gpu.deploy.dcgm-exporter": "true",
          "nvidia.com/gpu.deploy.device-plugin": "true",
          "nvidia.com/gpu.deploy.driver": "pre-installed",
          "nvidia.com/gpu.deploy.gpu-feature-discovery": "true",
          "nvidia.com/gpu.deploy.node-status-exporter": "true",
          "nvidia.com/gpu.deploy.operator-validator": "true",
          "nvidia.com/gpu.family": "ampere",
          "nvidia.com/gpu.machine": "System-Product-Name",
          "nvidia.com/gpu.memory": "24576",
          "nvidia.com/gpu.present": "true",
          "nvidia.com/gpu.product": "NVIDIA-GeForce-RTX-3090-SHARED",
          "nvidia.com/gpu.replicas": "2",
          "nvidia.com/mig.capable": "false",
          "nvidia.com/mig.strategy": "single"
        }
      },`
The same content was output for node7 and node8, while it was not output for other nodes.

**Comment by younghoon-klleon** [comment]
![kubelet_log](https://github.com/kubernetes/kubernetes/assets/102716628/9af55cdc-b7c1-4aff-a403-36683f424e22)
This is the log for kubelet.service.

**Comment by younghoon-klleon** [comment]
```
● kubelet.service - Kubernetes Kubelet Server
     Loaded: loaded (/etc/systemd/system/kubelet.service; enabled; vendor preset: enabled)
     Active: active (running) since Wed 2024-01-31 14:39:36 KST; 5h 3min ago
       Docs: https://github.com/GoogleCloudPlatform/kubernetes
   Main PID: 2039818 (kubelet)
      Tasks: 104 (limit: 308851)
     Memory: 187.5M
        CPU: 1h 1min 35.390s
     CGroup: /system.slice/kubelet.service
             └─2039818 /usr/local/bin/kubelet --v=2 --node-ip=${ip} --hostname-override=node1 --bootstrap-kubeconfig=/etc/kubernetes/bootstrap-kubelet.conf --config=/etc/kubernetes/kubelet-config.yaml --kubeconfig=/etc/kubernetes/kubelet.conf --container-runtime-endpoint=unix:///var/run/containerd/containerd.sock --runtime-cgroups=/system.slice/containerd.service

Jan 31 19:42:20 node1 kubelet[2039818]: I0131 19:42:20.570387 2039818 kubelet_getters.go:187] "Pod status updated" pod="kube-system/kube-scheduler-node1" status=Running
Jan 31 19:42:36 node1 kubelet[2039818]: I0131 19:42:36.679831 2039818 kubelet_getters.go:187] "Pod status updated" pod="kube-system/kube-apiserver-node1" status=Running
Jan 31 19:42:36 node1 kubelet[2039818]: I0131 19:42:36.679921 2039818 kubelet_getters.go:187] "Pod status updated" pod="kube-system/kube-scheduler-node1" status=Running
Jan 31 19:42:36 node1 kubelet[2039818]: I0131 19:42:36.679959 2039818 kubelet_getters.go:187] "Pod status updated" pod="kube-system/kube-controller-manager-node1" status=Running
Jan 31 19:42:50 node1 kubelet[2039818]: I0131 19:42:50.574693 2039818 kubelet_getters.go:187] "Pod status updated" pod="kube-system/kube-controller-manager-node1" status=Running
Jan 31 19:42:50 node1 kubelet[2039818]: I0131 19:42:50.574768 2039818 kubelet_getters.go:187] "Pod status updated" pod="kube-system/kube-apiserver-node1" status=Running
Jan 31 19:42:50 node1 kubelet[2039818]: I0131 19:42:50.574810 2039818 kubelet_getters.go:187] "Pod status updated" pod="kube-system/kube-scheduler-node1" status=Running
Jan 31 19:43:20 node1 kubelet[2039818]: I0131 19:43:20.570604 2039818 kubelet_getters.go:187] "Pod status updated" pod="kube-system/kube-scheduler-node1" status=Running
Jan 31 19:43:20 node1 kubelet[2039818]: I0131 19:43:20.570687 2039818 kubelet_getters.go:187] "Pod status updated" pod="kube-system/kube-controller-manager-node1" status=Running
Jan 31 19:43:20 node1 kubelet[2039818]: I0131 19:43:20.570732 2039818 kubelet_getters.go:187] "Pod status updated" pod="kube-system/kube-apiserver-node1" status=Running
```
This is the log for kubelet.service.

**Comment by ndixita** [comment]
Can you please provide the kubelet logs with level 4 for further debugging?

**Comment by younghoon-klleon** [comment]
solved after update kernel version to 5.15
